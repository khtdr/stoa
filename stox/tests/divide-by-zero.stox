#!/usr/bin/env stox

print 1 / 0;
