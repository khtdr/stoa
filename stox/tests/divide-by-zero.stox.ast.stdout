(program 
   (print (/ 1 0))
)
